internal/scorecard/alpha/tests/bundle_test.go

```
@@ -18,403 +18,397 @@ import (
 	"path/filepath"
 	"testing"
 
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 	apimanifests "github.com/operator-framework/api/pkg/manifests"
 	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	scapiv1alpha2 "github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha2"
 	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-
-	scapiv1alpha2 "github.com/operator-framework/operator-sdk/pkg/apis/scorecard/v1alpha2"
 )
 
-var testBundle = filepath.Join("..", "testdata", "bundle")
+var _ = Describe("Basic and OLM tests", func() {
+	var (
+		testBundle = filepath.Join("..", "testdata", "bundle")
+		res        scapiv1alpha2.ScorecardTestResult
+	)
+
+	BeforeEach(func() {
+		res = scapiv1alpha2.ScorecardTestResult{
+			Name:   "Scorecard result struct",
+			State:  scapiv1alpha2.PassState,
+			Errors: make([]string, 0),
+		}
+	})
 
-func TestBundleCRs(t *testing.T) {
-	cases := []struct {
-		bundlePath string
-		crCount    int
-	}{
-		{testBundle, 1},
-	}
+	Describe("Test Bundle CRs", func() {
+		var (
+			bundle  *apimanifests.Bundle
+			err     error
+			crCount int
+			crList  []unstructured.Unstructured
+		)
 
-	for _, c := range cases {
-		t.Run(c.bundlePath, func(t *testing.T) {
+		It("Check Bundle CRs", func() {
+			crCount = 1
 
-			bundle, err := apimanifests.GetBundleFromDir(c.bundlePath)
-			if err != nil {
-				t.Fatalf("Error getting bundle: %v", err)
-			}
+			bundle, err = apimanifests.GetBundleFromDir(testBundle)
+
+			Expect(err).ToNot(HaveOccurred())
 
-			var crList []unstructured.Unstructured
 			crList, err = GetCRs(bundle)
-			if err != nil {
-				t.Error(err)
-				return
-			}
-			if len(crList) != c.crCount {
-				t.Errorf("Wanted %d CRs but got: %d", c.crCount, len(crList))
-				return
-			}
+			Expect(crCount).To(Equal(len(crList)))
+			Expect(err).ToNot(HaveOccurred())
```

Nit: always check `err` before everything else as to not mask errors
```diff
crList, err = GetCRs(bundle)
-Expect(crCount).To(Equal(len(crList)))
Expect(err).ToNot(HaveOccurred())
+Expect(crCount).To(Equal(len(crList)))
```